### DISCUSSION TITLE
Sound notification

### DISCUSSION THREAD

**Discussion by NazarRecovery** [Chatterino/chatterino2#4488] [Ideas] [answered]
Your app works great, I recommend this app to my friends and colleagues, but they are missing some functionality. 
They have small communities and correspondingly little activity. They lack an **audible notification** when **any chat message** comes in. I wanted to find information on this but couldn't find it. 
Could you suggest how much of this is an implementable feature?

**Comment by Felanbird** [comment, marked as answer]
Add this as a highlight:
`.+`
![image](https://user-images.githubusercontent.com/41973452/228746915-103c87f0-c39a-4274-9aaa-5c38da3d720e.png)

**Reply by NazarRecovery** [reply]
It's working, thank you very much!

**Comment by NazarRecovery** [comment]
I'll try this
